added analytic formulas

alec-flexcompute opened the pull request.

alec-flexcompute committed:
added analytic formulas

e-g-melo reviewed (commented):
Hi @alec-flexcompute! Thanks for adding this content, it will help us a lot!

It seems there are some duplicate files and some automatically generated folders named `_checkpoints`. I wonder if they are intentional or if we should remove them.
<img width="260" height="533" alt="image" src="https://github.com/user-attachments/assets/6884f8b2-179f-4795-ad42-854a598107af" />

alec-flexcompute commented:
Hi @e-g-melo , those were generated by running compile_rst_categories.py inside the docs folder

FilipeFcp commented on _faqs/what-does-the-FluxMonitor-record.md:
```diff
@@ -0,0 +1,32 @@
+<center>
+    Re$(\sum\limits_{\text{cells in monitor}}\frac{1}{2}(E_1\cdot H_2^*-E_2\cdot H_1^*)dA_{\text{cell}})$
+</center><br>
+where $dA$ is the area of each cell in the monitor, and the field components 1 and 2 are given depending on the monitor geometry: for $x$-normal monitors, 1 is $y$ and 2 is $z$; for $y$-normal monitors, 1 is $x$ and 2 is $z$; for $z$-normal monitors, 1 is $x$ and 2 is $y$. See the code [here](/https://docs.flexcompute.com/projects/tidy3d/en/latest/_modules/tidy3d/components/data/monitor_data.html ...
```
Is the "/" before the link a HTML stuff? When I render this link in a notebook, it doesn't work. It only works without the first "/" -> [here](https://docs.flexcompute.com/projects/tidy3d/en/latest/_modules/tidy3d/components/

FilipeFcp commented on _faqs/what-does-the-FluxMonitor-record.md:
```diff
@@ -0,0 +1,32 @@
+The FluxMonitor records field data tangential $E$ and $H$ fields colocated to the cell boundaries in the monitor's 2D plane grid. It then computes and integrates the Poynting vector, returning the real part of this integral as the flux.
+<div> </div>
+<center>
+    Re$(\sum\limits_{\text{cells in monitor}}\frac{1}{2}(E_1\cdot H_2^*-E_2\cdot H_1^*)dA_{\text{cell}})$
```
What is the "Re" before the "$"?

FilipeFcp reviewed (approved):
Nice!
Just two minor comments. All the rest seems good!

alec-flexcompute committed:
fixed links

alec-flexcompute commented:
Thanks for the comments @FilipeFcp ! I fixed the links (I looked up how to insert a hyperlink in md and they for some reason told me to add a / at the beginning. I changed "Re" to "Real"

tomflexcompute reviewed (approved):
Thanks @alec-flexcompute !
